experimental/add_to_app/README.md

```diff
@@ -71,6 +83,50 @@ These apps showcase a relatively straightforward integration of
 If you are new to Flutter's add-to-app APIs, these projects are a great place
 to begin learning how to use them.
 
+### `android_using_plugin` and `ios_plugin`
+
+These apps are similar to `android_fullscreen` and `ios_fullscreen`, with the
+following differences:
+
+* Rather than importing `flutter_module`, they import
+  `flutter_module_using_plugin`.
+* They include the native code (Kotlin or Swift) required to initialize plugins
+  at Flutter engine creation time.
+* Their Flutter view includes an additional button that opens the Flutter docs
+  in the mobile device's browser.
+
+If you're interested in learning what additional steps an app needs to take in
+order to use a Flutter module that relies on plugins, these projects can help.
+
+### `android_using_prebuilt_module`
+
+This app is essentially identical to `android_fullscreen` with one key
+difference:
+
+* The Flutter module is *not* built automatically when the app builds. Instead,
+  it's built separately into an `aar`. The Android app is configured to import
+  that `aar` along with its other gradle dependencies.
+
+This can be useful for teams that don't want to require every developer working
+on the app to have the Flutter toolchain installed on their local machines.
+
+Prior to building `android_using_prebuilt_module` for the first time, the
+Flutter module needs to be built into an `aar`. You can do that by running this
+command from the `flutter_module` directory:
+
+```
+flutter build aar --debug
+```
+
+To build a release version of the `aar`, simply omit the debug flag:
+
+```
+flutter build aar
+```
+
```

Comment:
Add a line to say that both debug and release aars are hooked to be recognized in the outer android app